hack/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: build push
```

Comment:
Is there any requirement to use makefile for this? All our scripts are for now plain bash

Reply:
not really, I'm just following the pattern used in the AppSRE repos, but that's not required.

Reply:
@danielezonca do I need to change this or can we leave it for now?

Reply:
I think we should go for `bash` unless that's not supported by app-interface? We've already got a lot of bash in the project, so we could create a `build` or `app-interface` directory with specific scripts in there to support the builds.

Reply:
Ok let's keep it as a normal bash script for now and if necessary we can easily create the makefile.
@robpblake 
Is it fine for you?

Reply:
Yep, +1 to bash. There is a lot of bash skills on the team, so that helps us even more.